cargo-generate/remove_rprs.rhai

```diff
@@ -0,0 +1,9 @@
+file::delete("README.md");
+file::delete("debug_probes.md");
+file::delete("CODE_OF_CONDUCT.md");
+file::delete("MIT");
+file::delete("APACHE2.0");
```

Comment:
IANAL so I'm not 100% sure but doesn't `cargo generate` make the output a derivative work of this repo?
If so shouldn't the output be using the same (or at least compatible) licenses?

Reply:
I kinda started from the idea of "what does it look like once everything is removed".
Honestly I would prefer that this template was under a Public-Domain-equivalent license (BSD0 or CC0).
There's nothing novel or creative in here so it's unlikely to count as an Original Work, it's just the boilerplate that everyone would write or copy if they were building a project themselves.

That said, not what I was intending to push here so I'll remove those files from the delete list.

Reply:
If I correctly understand https://cargo-generate.github.io/cargo-generate/templates/conditional.html, it would be possible to select which license. cargo generate then puts the right file in place, I would love to see this applied here.